Failing for Windows tests

output.md file is failing for windows:

```
FAILED tests/test_cli.py::test_info[options1-example0] - UnicodeDecodeError: 'utf-8' codec can't decode byte 0x85 in position 4182: invalid start byte
FAILED tests/test_cli.py::test_info[options1-example1] - AssertionError: diff -ru --exclude __pycache__ --exclude .tool_cache --exclude .cache C:\a\sv-simpleparser\sv-simpleparser\tests\refdata\tests.test_cli\test_info\[adder.sv/output.md](http://adder.sv/output.md) C:
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -8,7 +8,7 @@ jobs:
         os:
           - "ubuntu-latest"
           - "macos-latest"
-          # FIXME: - "windows-latest"
+          - "windows-latest"
         python-version:
           - "3.10"
           - "3.11"

--- a/src/sv_simpleparser/_gen_templates.py
+++ b/src/sv_simpleparser/_gen_templates.py
@@ -48,15 +48,15 @@ def gen_instance(mod):
     return instance_file
 
 
-def gen_markdown_table(mod: Module) -> tuple[Table, Table]:
-    table = Table(title=f"`{mod.name}` Interface", box=box.MARKDOWN)
+def gen_markdown_table(mod: Module, width: int | None = None) -> tuple[Table, Table]:
+    table = Table(title=f"`{mod.name}` Interface", box=box.MARKDOWN, width=width)
 
     table.add_column("Name", no_wrap=True)
     table.add_column("Dimension", no_wrap=True)
     table.add_column("I/O", no_wrap=True)
     table.add_column("Functional Description")
 
-    table_param = Table(title=f"`{mod.name}` Parameters", box=box.MARKDOWN)
+    table_param = Table(title=f"`{mod.name}` Parameters", box=box.MARKDOWN, width=width)
 
     table_param.add_column("Name", no_wrap=True)
     table_param.add_column("Dimension", no_wrap=True)

--- a/src/sv_simpleparser/cli.py
+++ b/src/sv_simpleparser/cli.py
@@ -142,11 +142,18 @@ def gen_sv_instance(ctx, file_path):  # noqa: ARG001
 @cli.command()
 @arg_filepath
 @pass_ctx
-def info(ctx: Ctx, file_path: Path) -> None:
+@click.option(
+    "--width",
+    "-w",
+    type=int,
+    help="Enforce Table Width. Environment Variable SV_SIMPLEPARSER_WIDTH.",
+    envvar="SV_SIMPLEPARSER_WIDTH",
+)
+def info(ctx: Ctx, file_path: Path, width: int | None = None) -> None:
     """Outputs information about a SV file."""
     file = parse_file(file_path)
     for module in file.modules:
-        table_io, table_param = gen_markdown_table(module)
+        table_io, table_param = gen_markdown_table(module, width=width)
         ctx.console.print(table_param)
         ctx.console.print(table_io)
 

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -52,8 +52,4 @@ def runner() -> CliRunner:
 @fixture(autouse=True)
 def enforce_terminal_size(monkeypatch):
     """Fix issue with varying terminal size."""
-
-    def get_terminal_size(*args, **kwargs):
-        return (120, 60)
-
-    monkeypatch.setattr("shutil.get_terminal_size", get_terminal_size)
+    monkeypatch.setenv("SV_SIMPLEPARSER_WIDTH", "75")

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -47,3 +47,13 @@ def examples() -> Path:
 def runner() -> CliRunner:
     """CLI Runner."""
     return CliRunner()
+
+
+@fixture(autouse=True)
+def enforce_terminal_size(monkeypatch):
+    """Fix issue with varying terminal size."""
+
+    def get_terminal_size(*args, **kwargs):
+        return (120, 60)
+
+    monkeypatch.setattr("shutil.get_terminal_size", get_terminal_size)

--- a/tests/refdata/tests.test_cli/test_info/adder.sv/output.md
+++ b/tests/refdata/tests.test_cli/test_info/adder.sv/output.md
@@ -1,19 +1,20 @@
-                   `adder` Parameters                    
-                                                         
-| Name           | Dimension | Functional Description   |
-|----------------|-----------|--------------------------|
-| `DATA_WIDTH`   |           | Width of input operands  |
-| `OUTPUT_WIDTH` |           | Test configuration value |
-                                                         
-                               `adder` Interface                                
-                                                                                
-| Name          | Dimension          | I/O      | Functional Description       |
-|---------------|--------------------|----------|------------------------------|
-| `A`           | `[DATA_WIDTH-1:0]` | `input`  | Packed input operand A       |
-| `B`           | `[DATA_WIDTH-1:0]` | `input`  | Packed input operand B       |
-| `X`           | `[DATA_WIDTH:0]`   | `output` | Packed sum output            |
-| `byte_p`      | `[7:0]`            | `input`  | Packed byte input            |
-| `word_p`      | `[3:0][7:0]`       | `input`  | Packed 32-bit word (4 bytes) |
-| `flag_u`      | `1`                | `input`  | Unpacked single bit          |
-| `arr_u [0:3]` | `[7:0]`            | `input`  | Unpacked byte array          |
-                                                                                
+                            `adder` Parameters                             
+                                                                           
+| Name                 | Dimension     | Functional Description           |
+|----------------------|---------------|----------------------------------|
+| `DATA_WIDTH`         |               | Width of input operands          |
+| `OUTPUT_WIDTH`       |               | Test configuration value         |
+                                                                           
+                             `adder` Interface                             
+                                                                           
+| Name          | Dimension          | I/O      | Functional Description  |
+|---------------|--------------------|----------|-------------------------|
+| `A`           | `[DATA_WIDTH-1:0]` | `input`  | Packed input operand A  |
+| `B`           | `[DATA_WIDTH-1:0]` | `input`  | Packed input operand B  |
+| `X`           | `[DATA_WIDTH:0]`   | `output` | Packed sum output       |
+| `byte_p`      | `[7:0]`            | `input`  | Packed byte input       |
+| `word_p`      | `[3:0][7:0]`       | `input`  | Packed 32-bit word (4   |
+|               |                    |          | bytes)                  |
+| `flag_u`      | `1`                | `input`  | Unpacked single bit     |
+| `arr_u [0:3]` | `[7:0]`            | `input`  | Unpacked byte array     |
+                                                                           

--- a/tests/refdata/tests.test_cli/test_info/bcd_adder.sv/output.md
+++ b/tests/refdata/tests.test_cli/test_info/bcd_adder.sv/output.md
@@ -1,25 +1,25 @@
-           `bcd_adder` Parameters            
-                                             
-| Name | Dimension | Functional Description |
-|------|-----------|------------------------|
-                                             
-                  `bcd_adder` Interface                   
-                                                          
-| Name   | Dimension | I/O      | Functional Description |
-|--------|-----------|----------|------------------------|
-| `a`    | `[3:0]`   | `input`  |                        |
-| `b`    | `[3:0]`   | `input`  |                        |
-| `cin`  | `1`       | `input`  |                        |
-| `sum`  | `[3:0]`   | `output` |                        |
-| `cout` | `1`       | `output` |                        |
-                                                          
-          `tb_bcdadder` Parameters           
-                                             
-| Name | Dimension | Functional Description |
-|------|-----------|------------------------|
-                                             
-              `tb_bcdadder` Interface              
-                                                   
-| Name | Dimension | I/O | Functional Description |
-|------|-----------|-----|------------------------|
-                                                   
+                          `bcd_adder` Parameters                           
+                                                                           
+| Name      | Dimension         | Functional Description                  |
+|-----------|-------------------|-----------------------------------------|
+                                                                           
+                           `bcd_adder` Interface                           
+                                                                           
+| Name      | Dimension     | I/O         | Functional Description        |
+|-----------|---------------|-------------|-------------------------------|
+| `a`       | `[3:0]`       | `input`     |                               |
+| `b`       | `[3:0]`       | `input`     |                               |
+| `cin`     | `1`           | `input`     |                               |
+| `sum`     | `[3:0]`       | `output`    |                               |
+| `cout`    | `1`           | `output`    |                               |
+                                                                           
+                         `tb_bcdadder` Parameters                          
+                                                                           
+| Name      | Dimension         | Functional Description                  |
+|-----------|-------------------|-----------------------------------------|
+                                                                           
+                          `tb_bcdadder` Interface                          
+                                                                           
+| Name     | Dimension       | I/O    | Functional Description            |
+|----------|-----------------|--------|-----------------------------------|
+                                                                           

--- a/tests/refdata/tests.test_cli/test_info/param_module.sv/output.md
+++ b/tests/refdata/tests.test_cli/test_info/param_module.sv/output.md
@@ -1,36 +1,37 @@
-                `param_module` Parameters                 
-                                                          
-| Name             | Dimension | Functional Description  |
-|------------------|-----------|-------------------------|
-| `WIDTH`          |           | Width of the input data |
-| `DEPTH`          |           |                         |
-| `INIT_VAL`       | `[7:0]`   |                         |
-| `ENABLE_FEATURE` |           |                         |
-                                                          
-                     `param_module` Interface                      
-                                                                   
-| Name        | Dimension     | I/O      | Functional Description |
-|-------------|---------------|----------|------------------------|
-| `clk`       | `1`           | `input`  |                        |
-| `rst_n`     | `1`           | `input`  | active-low reset       |
-| `data_in`   | `[WIDTH-1:0]` | `input`  | Input data             |
-| `data_out`  | `[WIDTH-1:0]` | `output` |                        |
-| `bidir_bus` | `[DEPTH-1:0]` | `inout`  |                        |
-                                                                   
-               `sub_module` Parameters               
-                                                     
-| Name         | Dimension | Functional Description |
-|--------------|-----------|------------------------|
-| `DATA_WIDTH` |           |                        |
-| `INIT_VALUE` | `[7:0]`   |                        |
-                                                     
-                           `sub_module` Interface                           
-                                                                            
-| Name          | Dimension            | I/O      | Functional Description |
-|---------------|----------------------|----------|------------------------|
-| `clk`         | `1`                  | `input`  |                        |
-| `reset`       | `1`                  | `input`  |                        |
-| `input_data`  | `[DATA_WIDTH-1:0]`   | `input`  |                        |
-| `output_data` | `[DATA_WIDTH-1:0]`   | `output` |                        |
-| `config_bus`  | `[DATA_WIDTH/2-1:0]` | `inout`  |                        |
-                                                                            
+                         `param_module` Parameters                         
+                                                                           
+| Name                   | Dimension     | Functional Description         |
+|------------------------|---------------|--------------------------------|
+| `WIDTH`                |               | Width of the input data        |
+| `DEPTH`                |               |                                |
+| `INIT_VAL`             | `[7:0]`       |                                |
+| `ENABLE_FEATURE`       |               |                                |
+                                                                           
+                         `param_module` Interface                          
+                                                                           
+| Name          | Dimension       | I/O        | Functional Description   |
+|---------------|-----------------|------------|--------------------------|
+| `clk`         | `1`             | `input`    |                          |
+| `rst_n`       | `1`             | `input`    | active-low reset         |
+| `data_in`     | `[WIDTH-1:0]`   | `input`    | Input data               |
+| `data_out`    | `[WIDTH-1:0]`   | `output`   |                          |
+| `bidir_bus`   | `[DEPTH-1:0]`   | `inout`    |                          |
+                                                                           
+                          `sub_module` Parameters                          
+                                                                           
+| Name                | Dimension      | Functional Description           |
+|---------------------|----------------|----------------------------------|
+| `DATA_WIDTH`        |                |                                  |
+| `INIT_VALUE`        | `[7:0]`        |                                  |
+                                                                           
+                          `sub_module` Interface                           
+                                                                           
+|               |                      |          | Functional            |
+| Name          | Dimension            | I/O      | Description           |
+|---------------|----------------------|----------|-----------------------|
+| `clk`         | `1`                  | `input`  |                       |
+| `reset`       | `1`                  | `input`  |                       |
+| `input_data`  | `[DATA_WIDTH-1:0]`   | `input`  |                       |
+| `output_data` | `[DATA_WIDTH-1:0]`   | `output` |                       |
+| `config_bus`  | `[DATA_WIDTH/2-1:0]` | `inout`  |                       |
+                                                                           

--- a/tests/test_cli.py
+++ b/tests/test_cli.py
@@ -26,11 +26,13 @@ from pathlib import Path
 
 from click.testing import CliRunner
 from pytest import mark
-from test2ref import assert_refdata
+from test2ref import assert_refdata, configure
 
 from sv_simpleparser.cli import cli
 
 from .conftest import EXAMPLES_PATH
+
+configure(ignore_spaces=True)
 
 # We are just testing a reduced set of examples here
 EXAMPLES = (

